docs/building.md

```diff
@@ -222,27 +222,66 @@ cargo run --release --bin slint-viewer -- examples/printerdemo/ui/printerdemo.sl
 
 ## Generating the Documentation
 
-### Rust
+The Slint documentation consists of four parts:
 
-The documentation for the different crates is built using rustdoc.
+- The Slint tutorials
+- The Slint Rust DSL documentation
```

Comment:
Maybe this is where the confusion arises. There are five parts :-)

 - The Slint tutorials (built using mdbook)
 - The Slint Rust API documentation (built using rustic)
 - The Slint C++ API documentation (built using sphinx/exhale/breathe/doxygen/myst_parser)
 - The Slint Node.js API documentation (built using typed)
 - The Slint DSL documentation (built using sphinx with myst_parser for markdown)

Reply:
Ahhh, yeah I think the Rust part was where I got lost, well it doesn't fix my confusion, more adds, but will update!